operating_unit/security/operating_unit_security.xml

```diff
@@ -27,7 +27,7 @@
     </record>
     <record id="ir_rule_operating_unit_allowed_operating_units" model="ir.rule">
         <field name="model_id" ref="model_operating_unit" />
-        <field name="domain_force">[('id','in',user.operating_unit_ids.ids)]</field>
+        <field name="domain_force">[('id','in',operating_unit_ids)]</field>
```

Comment:
Why is `user` removed?

Reply:
The keyword "operating_unit_ids" is now in the context of the ir.rules
The idea is to prepare the ground for a potential widget selector (like the company one) and to have a method governing the whole thing. 

You can see the changes in ir_rule.py and res_users.py (def operating_units)